visualizations/frontend/other/TTable.vue

```diff
@@ -307,10 +330,14 @@ export default {
       showSpinner: true,
       displayRows: [],
       pagerResetFunction: null,
-      internalColumns: [],
+      filterableColumnsToShow: [],
+      additionalFilters: {},
     };
   },
   computed: {
+      isPdf(){
+          return window.location.href.includes('/pdf?')
+      },
```

Comment:
How do you feel about making this a prop instead of a computed property?

Relying a path on the window url makes the implementation tied to the way the PDF is currently implemented. Which makes it harder to update the implementation of pdf, without affecting this component. I also wonder how this might play with the iframe? Maybe Emi could shed some light here

If the CSV export and the table are very coupled by design (and we can't separate them for now), surfacing a prop (`canDownloadCSV`, for example) makes what the component will display more explicit and independently controlled at the point of use. We can default it to 'false', for the PDF implementation not to break.